src/main/resources/emails/deactivationWarning.tflh

```diff
@@ -0,0 +1,38 @@
+<html>
+<style type="text/css">
+body {
+	font-family: Arial, Helvetica, SourceSansPro-Regular;
+	color: #000000;
+	font-size: 11px
+}
+h1 {
+	font-size: 20px
+}
+h2 {
+	font-size: 16px;
+}
+
+a {
+	color:#0072BD;
+}
+</style>
+<body>
+<p style="color:red; font-size:12px; font-weight: bold;" align="center"><i>Classification: ${SECURITY_BANNER_TEXT}</i></p>
+
+<#assign name = person.getName()>
+Hi ${name},
+
+<#assign endDate = person.getEndOfTourDate()>
+<#assign daysRemaining = daysUntilEndOfTour>
+<p>Your end of tour date is planned on ${endDate}, at which point your ANET account will be deactivated (${daysRemaining} days remaining).</p>
```

Comment:
Please observe the proper date formatting of `endDate` (see other FreeMarker templates for examples).